isofit/radiative_transfer/radiative_transfer_engine.py

```diff
@@ -22,6 +22,7 @@
 import logging
 import os
 import time
+from itertools import product
```

Comment:
@pgbrodrick The `itertools.product` function is not used anymore. You could remove the import statement here.